source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRContainerRenderer.mm

```diff
@@ -55,11 +109,19 @@ - (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup
 
     container.frame = viewGroup.frame;
 
-    [ACRRenderer render:container
-               rootView:rootView
-                 inputs:inputs
-          withCardElems:containerElem->GetItems()
-          andHostConfig:acoConfig];
+    if(flowContainer != nil)
+    {
+        [container addArrangedSubview:flowContainer];
+    }
+    else if (gridLayout != nil) {
+        [container addArrangedSubview:gridLayout];
+    } else {
+        [ACRRenderer render:container
+                   rootView:rootView
+                     inputs:inputs
```

Comment:
here and everywhere - else and else if on new line and opening braces on a new line.